bundler/spec/commands/newgem_spec.rb

```diff
@@ -588,6 +588,22 @@ def create_temporary_dir(dir)
       expect(bundled_app("#{gem_name}/lib/#{require_path}.rb").read).to match(/class Error < StandardError; end$/)
     end
 
+    it "does not include git related files in files" do
+      bundle "gem #{gem_name}"
+
+      bundler_gemspec = Bundler::GemHelper.new(gemspec_dir).gemspec
+
+      expect(bundler_gemspec.file).to eq([])
```

Comment:
Is that meant to read like this?

```suggestion
      expect(bundler_gemspec.files).to eq([])
```